scipy/spatial/transform/rotation.py

```diff
@@ -1463,6 +1464,52 @@ def magnitude(self):
         else:
             return angles
 
+    def reduce(self, left=None, right=None):
+        """Reduce this rotation with the provided rotation groups.
+
+        Parameters
+        ----------
+        left : `Rotation` instance
+            Object containing the left rotation group.
+        right : `Rotation` instance
+            Object containing the right rotation group.
+
+        Returns
+        -------
+        reduced : `Rotation` instance
+            Object containing reduced rotations.
+        """
+        if left is None and right is None:
+            raise ValueError("Reduction requires either a left or right group")
```

Comment:
I think in this case returning `self.copy()` is totally fine.

Reply:
Nice idea.  I have changed it.